specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-10-01/examples/ChildAvailabilityStatus_GetByResource.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "resourceUri": "resourceUri",
+    "api-version": "2022-10-01",
+    "$expand": "recommendedactions"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/227b734f-e14f-4de6-b7fc-3190c21e69f6/resourceGroups/JUHACKETRHCTEST/providers/Microsoft.Compute/virtualMachineScaleSets/rhctest/virtualMachines/4/providers/Microsoft.ResourceHealth/child/availabilityStatuses/current",
+        "name": "current",
+        "type": "Microsoft.ResourceHealth/AvailabilityStatuses",
+        "location": "southcentralus",
+        "properties": {
+          "availabilityState": "Available",
+          "summary": "There aren't any known Azure platform problems affecting this virtual machine",
+          "detailedStatus": "",
+          "reasonType": "",
+          "occuredTime": "2018-11-30T23:37:44Z",
+          "reasonChronicity": "Persistent",
+          "reportedTime": "2018-12-04T01:19:11.955255Z",
+          "recommendedActions": [
```

Comment:
>recommendedActions [](http://example.com/codeflow?start=11&length=18)

why isn't this propertyname recommendedActions camelCased?

Reply:
to give context this is an existing endpoint that has apparently been missing from the swagger in later versions. we can't change the name without a breaking change